azerothcore/azerothcore-wotlk
# Fix(Core/Ulduar): Fixed the Stone Nova stacking issue in 25man mode

## CHANGES PROPOSED:
-  Added spellID 63978 to the spells that should stack 

## ISSUES ADDRESSED:
- Closes #3564 

## TESTS PERFORMED:
- Tested at Linux. 
- Spawned two rubbles in the Ulduar 25man mode 
- Debuff were stacking properly from both Rubbles

## HOW TO TEST THE CHANGES:
As the author of #3564 wrote:
- Go to Ulduar 25.
- npc add 33768 (create at least 2)
- Start combat and observe.

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
